data/transaction.go

```diff
@@ -27,8 +27,8 @@ type Transaction struct {
 	Value     string `json:"value"`
 	RcvAddr   string `json:"receiver"`
 	SndAddr   string `json:"sender"`
-	GasPrice  uint64 `json:"gasPrice,omitempty"`
-	GasLimit  uint64 `json:"gasLimit,omitempty"`
+	GasPrice  uint64 `json:"gasPrice"`
```

Comment:
this struct was not alined with mx-chain-go's `FrontendTransaction` ( https://github.com/multiversx/mx-chain-core-go/blob/main/data/transaction/frontendTransaction.go#L4 )